Learn_CRM_Sdk/CustomViewsViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CustomViewsViewController.swift
+//  Learn_CRM_Sdk
+//
+//  Created by ZU-IOS on 10/03/20.
+//  Copyright © 2020 ZU-IOS. All rights reserved.
+//
+
+import UIKit
+import ZCRMiOS
+
+protocol customViewDelegate : class{
+    func changeToCustomView(_ leadStatus : String , _ cvRow : Int)
+}
+
+class CustomViewsViewController: UIViewController {
+
+    let customViewTableView : UITableView = UITableView()
+    var customViews : [ZCRMCustomView] = [ZCRMCustomView]()
+
+
+    var module_Name : String!
```

Comment:
Why do you need a module name in the custom view controller?

Reply:
module_Name no need na